### ISSUE TITLE
Coordinates lost in URL

### ISSUE THREAD

**Issue by DenisCarriere** [osmottawa/CrowdiD#2]
Since you're using an iFrame (never use iFrames), you are losing all the metadata that the iD editor is pushing to the URL (lat, lng, zoom).

This is very useful when you want to give someone a link with those exact coordinates.

![image](https://cloud.githubusercontent.com/assets/550895/19286169/aba22464-8fca-11e6-83f8-5c92bb3eded3.png)

![image](https://cloud.githubusercontent.com/assets/550895/19286202/ca648d42-8fca-11e6-9977-95939acb0100.png)

**Comment by Monduiz** [comment]
I agree. I will look into changing that to a standard frame after I have resolved the points of interest and the popup.

**Comment by DenisCarriere** [comment]
@Monduiz You got lots of work on your plate ;)

**Comment by Monduiz** [comment]
One step at a time!
